test/celeritas/optical/Cerenkov.test.cc

```diff
@@ -307,12 +316,7 @@ TEST_F(CerenkovTest, TEST_IF_CELERITAS_DOUBLE(generator))
     };
 
     size_type num_samples = 64;
-
-    OpticalDistributionData dist;
-    dist.time = 0;
-    dist.charge = charge;
-    dist.material = material;
-    dist.points[StepPoint::pre].pos = {0, 0, 0};
+    OpticalStepCollectorData step_collector;
```

Comment:
I'd rename this `step` because it's the collected data over a step, it doesn't do any collecting. Same with above.